openmdao/core/tests/test_group.py

```diff
@@ -514,6 +514,54 @@ def compute(self, inputs, outputs):
                          "Group (<model>): src_indices has been defined in both "
                          "connect('indep.x', 'C1.x') and add_input('C1.x', ...).")
 
+    def test_incompatible_src_indices_error(self):
+        class ControlInterpComp(om.ExplicitComponent):
+
+            def setup(self):
+                self.add_output('x', shape=(3, 1))
+
+            def compute(self, inputs, outputs):
```

Comment:
don't really need to redefine these `compute`/`compute_partials` methods, `pass` is the default implementation